packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -90,14 +90,14 @@ export class MessageWidget extends Component {
    */
   static getAvatarsFromParticipants(props) {
```

Comment:
Should we rename this to something like `getAvatarsFromActivityActors`? or just `FromActors`?

Reply:
yeah, i meant to rename it but got distracted. have a preference between those two?